stories/urban-heating.stories.mdx

```diff
@@ -13,8 +13,21 @@ taxonomy:
   - name: Topics
     values:
       - Environmental Justice
+      - Urban
+      - Heat
+  - name: Source
+    values:
+      - Community Contributed
 ---
 
+<Block> 
+  <Prose>
+
+    ###### This story is part of a study conducted on both heat and pollution stress in the Houston Metropolitan Area. [The data story that higlights pollution stress can be found here.](https://www.earthdata.nasa.gov/dashboard/stories/houston-aod)
```

Comment:
for internal links use `<Link>` component
https://github.com/US-GHG-Center/veda-config-ghg/pull/572/commits/f91b4041e359d15daea54e78b2fa01f4190c7af6